robolectric/src/test/java/org/robolectric/shadows/ShadowNotificationManagerTest.java

```diff
@@ -93,4 +97,32 @@ public void testCancelAll() throws Exception {
     assertNull(shadowOf(notificationManager).getNotification(null, 1));
     assertNull(shadowOf(notificationManager).getNotification(null, 31));
   }
+
+  @Test
+  public void testGetActiveNotifications() throws Exception {
```

Comment:
Done. Also added minSdk to the shadow implementation's @Implementation annotation.